packages/core/src/components/context-menu/contextMenuTarget.tsx

```diff
@@ -33,6 +35,8 @@ export function ContextMenuTarget<T extends { prototype: IContextMenuTarget }>(c
             // always return `element` in case caller is distinguishing between `null` and `undefined`
             return element;
         }
+        /* tslint:disable:no-this-assignment */
+        const thisElem = this;
```

Comment:
Doesn't the fat-arrow definition on `onContextMenu` maintain the same `this` binding as this parent scope? Seems like you don't need this.

Reply:
yeah I don't think this is necessary at all.

@mcintyret please try removing this line and just using `this` directly below.